{{ cookiecutter.__package_name_kebab_case }}/.gitlab-ci.yml

```diff
@@ -67,16 +68,14 @@ Compute Image Hash:
         - pyproject.toml
 
 # Build CI Docker image.
-Build CI:
+Build CI image:
   extends:
     - .docker
   stage: build
   variables:
     DOCKER_IMAGE: $CI_REGISTRY_IMAGE/ci
+    DOCKER_IMAGE_SHA: $CI_IMAGE_SHA
```

Comment:
@sinopeus This change should also be applied downstream to forks of Poetry Cookiecutter. Otherweise, the value of this env var is missing in the `.docker` section.

Reply:
Good to know! We'll resolve it downstream.